pkg/bootstrap/docker/openshift/metrics.go

```diff
@@ -65,11 +65,35 @@ func (h *Helper) InstallMetrics(f *clientcmd.Factory, hostName, imagePrefix, ima
 		return errors.NewError("cannot create metrics deployer secret").WithCause(err).WithDetails(h.OriginLog())
 	}
 
-	// Create deployer Pod
+	// Create the job client
+	jobClient := kubeClient.Batch().Jobs(infraNamespace)
+
+	// Create the job
 	deployerPod := metricsDeployerPod(hostName, imagePrefix, imageVersion)
```

Comment:
you should rename the function to metricsDeployerJob and return a Job instead of a Pod